export_spdx/data.py

```diff
@@ -131,14 +131,13 @@ def get_bom_components(verdict, exclude_ignored=False):
     # if 'components' not in res:
     if True:
         # Getting the component list via a request is much quicker than the new Client model
-        thishref = res['href'] + "/components?limit=5000"
-        headers = {
-            'accept': "application/vnd.blackducksoftware.bill-of-materials-6+json",
-        }
-        res = globals.bd.get_json(thishref, headers=headers)
-        bom_comps = res['items']
-    # else:
-    #     bom_comps = globals.bd.get_resource('components', parent=ver)
+        # thishref = res['href'] + "/components?limit=5000"
+        # headers = {
+        #     'accept': "application/vnd.blackducksoftware.bill-of-materials-6+json",
+        # }
+        # res = globals.bd.get_json(thishref, headers=headers)
+        # bom_comps = res['items']
+        bom_comps = globals.bd.get_resource('components', parent=verdict)
     for comp in bom_comps:
```

Comment:
I do not understand the reason for this change. Please add comment explaining the reason.

Reply:
This part of the script (in particular the get_json function) caused an error on my local machine. From what I could understand, both get_json and get_resource fetch BOM-components, but according to a comment in data.py, using get_json is faster. But due to get_json causing an error, I decided to use get_resource to get the script running.

Reply:
Ok. Please add an according comment to the source code. Otherwise no one will comprehend why this is different from the original code.

Reply:
Alright, I added the comment to data.py